bgpd/bgp_packet.c

```diff
@@ -3712,10 +3712,6 @@ static int bgp_capability_msg_parse(struct peer *peer, uint8_t *pnt,
 			zlog_err("%pBP: Capability length error", peer);
 			bgp_notify_send(peer->connection, BGP_NOTIFY_CEASE,
 					BGP_NOTIFY_SUBCODE_UNSPECIFIC);
-			/*
```

Comment:
I originally left the pnt addition in because if we ever modified the code I wanted a reminder that the length pointer needed to be handled appropriately.  Why are we removing this commented out code?